src/commands/shell.mjs

```diff
@@ -21,11 +23,10 @@ async function shellCommand(argv) {
 
   /** @type {import('node:repl').ReplOptions} */
   const replArgs = {
-    prompt: `${argv.db_path || ""}> `,
+    prompt: `${argv.database ? argv.database : ""}> `,
     ignoreUndefined: true,
-    preview: argv.apiVersion !== "10",
-    // TODO: integrate with fql-analyzer for completions
-    completer: argv.apiVersion === "10" ? () => [] : undefined,
+    preview: undefined,
+    completer: undefined,
```

Comment:
these can still support v4 if included/fixed, see one of @henryfauna's open PRs.

Reply:
Ah, I'll let his in first, and then resolve the conflicts so we don't lose his work.

Reply:
This has been put back in.